### ISSUE TITLE
When I indent it goes to the wrong parent rem

### ISSUE THREAD

**Issue by Eikosa** [remnoteio/remnote-issues#939]
https://user-images.githubusercontent.com/20538090/110033650-b86af000-7d4a-11eb-9847-94a0cd982e9e.mp4

**Comment by marcothesis** [comment]
Yes, I encounter this issue sometimes too but not sure how to reproduce.
